ras_party/controllers/account_controller.py

```diff
@@ -746,6 +746,15 @@ def put_email_verification(token, tran, session):
         # We set the user as verified on the OAuth2 server.
         set_user_verified(email_address)
 
+    if not respondent.pending_email_address and respondent.password_verification_token:
+        if not respondent.password_verification_token:
```

Comment:
Is this reachable code? As the above means respondent.password_verification_token has to be True